examples/rvc-app/rvc-common/include/rvc-device.h

```diff
@@ -4,14 +4,16 @@
 #include "rvc-operational-state-delegate.h"
 #include <app/clusters/mode-base-server/mode-base-server.h>
 #include <app/clusters/operational-state-server/operational-state-server.h>
+#include <app/clusters/service-area-server/service-area-server.h>
+#include <app/clusters/service-area-server/service-area-delegate.h>
 
 #include <string>
 
 namespace chip {
 namespace app {
 namespace Clusters {
 
-class RvcDevice
+class RvcDevice :  public ServiceArea::Delegate
```

Comment:
```suggestion
class RvcDevice : public ServiceArea::Delegate
```

nit: one space vs two?